resources/netregexes.ts

```diff
@@ -108,7 +129,7 @@ export default class NetRegexes {
   /**
    * matches: https://github.com/quisquous/cactbot/blob/main/docs/LogGuide.md#14-networkstartscasting
    */
-  static startsUsing(params?: Params<typeof startsUsingParams[number]>): RegExp {
+  static startsUsing(params?: Params<StartsUsingParams>): Return<StartsUsingParams> {
```

Comment:
I meant to say this earlier, but this is really great.  Thanks for making this all so generic.  (I learned a bunch about typescript that I didn't know!)